package.json

```
@@ -31,7 +31,7 @@
   "scripts": {
     "prepare": "npm run build && mjs-entry src/index.ts > lib/index.mjs",
     "build": "tsc --build docs/examples",
-    "build:docs": "typedoc",
+    "build:docs": "typedoc src/index.ts --theme minimal",
```

Let's discuss the theme later, once we can actually use the docs in prod.
```suggestion
    "build:docs": "typedoc src/index.ts",
```